lib/core/text/flat.nit

```diff
@@ -84,6 +84,82 @@ redef class FlatText
 		return ns_i
 	end
 
+	# By escaping `self` to C, how many more bytes will be needed ?
+	#
+	# This enables a double-optimization in `escape_to_c` since if this
+	# method returns 0, then `self` does not need escaping and can be
+	# returned as-is
+	protected fun chars_to_escape_to_c: Int do
+		var its = _items
+		var max = last_byte
+		var pos = first_byte
+		var req_esc = 0
+		while pos <= max do
+			var c = its[pos]
+			if c == 0x0Au8 then
+				req_esc += 1
+			else if c == 0x09u8 then
+				req_esc += 1
+			else if c == 0x22u8 then
+				req_esc += 1
+			else if c == 0x27u8 then
+				req_esc += 1
+			else if c == 0x5Cu8 then
+				req_esc += 1
+			else if c < 32u8 then
```

Comment:
agree, you should explain what are these char in comment, not all people know their ASCII table by heart.